meta-lmp-bsp/dynamic-layers/freescale-layer/recipes-bsp/imx-mkimage/imx-boot_%.bbappend

```diff
@@ -1,6 +1,13 @@
 DEPENDS_remove = "optee-os"
 DEPENDS += "${@bb.utils.contains('MACHINE_FEATURES', 'optee', 'virtual/optee-os', '', d)}"
 
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+SRC_URI_append_imx8mmevk = " \
```

Comment:
This seems generic to the soc, so use 'append_mx8mm' instead (that way all other boards based on mx8mm will be able to reuse the same implementation).